draft-ietf-quic-tls.md

```diff
@@ -1058,6 +1058,10 @@ validation was not requested originally.  In such cases, the cookie extension
 could either be absent or it could indicate that an address validation token is
 not present.
 
+A server can use the cookie extension to store all state necessary to continue
+the connection.  This allows a server to avoid committing state for clients that
+have unvalidated source addresses.
+
```

Comment:
This works, though I think you should be clearer about how you pickle that information (self-encrypted AEAD).

However, if we're in the business of telling people what to do, we actually should document both this technique and the CH...SFIN, NST, RST technique that davidben and victor described.